extensions/browsers-profiles/README.md

```diff
@@ -0,0 +1,4 @@
+# Raycast Browsers Profiles
+> Access your browsers profiles from Raycast
+
+More info: https://github.com/skydiver/raycast-browsers-profiles
```

Comment:
there aren't any additional information there, let's just remove it

Reply:
i posted the extension repo so i can create a guide on how to add more browsers ...

anyway, changes are ready